pages/v2/signUp.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Box,
+  Flex,
+  Stack,
+  Heading,
+  Text,
+  Container,
+  Input,
+  Button,
+  SimpleGrid,
+  Avatar,
+  AvatarGroup,
+  useBreakpointValue,
+  IconProps,
+  Icon,
+  InputGroup,
+  InputLeftAddon,
+  FormControl,
+  FormLabel,
+  Select,
+  FormHelperText,
+  VStack,
+  HStack,
+} from '@chakra-ui/react';
+
+const colleges = [
+  { id: 1, name: 'University of Illinois at Urbana-Champaign' },
+  { id: 2, name: 'University of Illinois at Chicago' },
+  { id: 3, name: 'University of Illinois at Springfield' },
+  { id: 4, name: 'St. Louis University' },
+  { id: 5, name: 'Massachusetts Institute of Technology' },
+];
+
+export default function JoinOurTeam() {
+  const BPL = useBreakpointValue({ base: 'md', md: 'lg' });
+  return (
+    <HStack
+      bg={'gray.50'}
```

Comment:
No need for `{}` when you are just assigning a string literal to a prop.
Make this change for all occurrences.